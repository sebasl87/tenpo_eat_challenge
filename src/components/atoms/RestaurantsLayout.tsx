import React from 'react'
import styled from 'styled-components/native';

export interface IRestaurantsLayout {
    children: React.ReactNode;
}
const Layout = styled.View`
    height: 100%;
    background: #fff;
    padding: 42px 0px 0px 16px;
    border-top-left-radius: 28px;
    border-top-right-radius: 28px;
`
const RestaurantsLayout: React.FC<IRestaurantsLayout> = ({ children }) => {
    return (
        <Layout>
            {children}
        </Layout>
    );
}

export default RestaurantsLayout;