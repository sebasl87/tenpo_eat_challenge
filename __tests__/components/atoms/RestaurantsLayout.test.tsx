import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import RestaurantsLayout from "../../../src/components/atoms/RestaurantsLayout";


describe("RestaurantsLayout component", () => {
    it("should render successfully with RestaurantsLayout", () => {
        const restaurantsLayout = renderer
            .create(<RestaurantsLayout children="RestaurantsLayout_mock" />)
            .toJSON();

        expect(restaurantsLayout).toMatchSnapshot();
    });
});
