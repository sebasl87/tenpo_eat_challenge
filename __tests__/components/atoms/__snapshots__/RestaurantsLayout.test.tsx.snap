// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantsLayout component should render successfully with RestaurantsLayout 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
        "borderColor": "#00000029",
        "borderStyle": "solid",
        "borderTopLeftRadius": 28,
        "borderTopRightRadius": 28,
        "borderWidth": 1,
        "height": "100%",
        "paddingBottom": 0,
        "paddingLeft": 16,
        "paddingRight": 0,
        "paddingTop": 42,
      },
    ]
  }
>
  RestaurantsLayout_mock
</View>
`;
